src/controllers/admin/articles/index.js

```diff
@@ -40,17 +41,22 @@ router.post('/:id/edit', function(req, res) {
       content: req.fields.content,
       parent: req.fields.parent,
     },
-
   };
 
+  if (!mongoose.Types.ObjectId.isValid(req.fields.parent)) {
```

Comment:
checks if it's a valid id in the database. There were cases when `parent` was the string "null"